library_generation/test/integration_tests.py

```diff
@@ -45,6 +47,24 @@
 
 
 class IntegrationTest(unittest.TestCase):
+    def test_get_commit_message_success(self):
```

Comment:
I added this as an integration test because it required network connection for git clone and checkout of googleapis repository.